### DISCUSSION TITLE
建议美化这个地方嘻嘻强迫症的福音

### DISCUSSION THREAD

**Discussion by LOVE-END** [the1812/Bilibili-Evolved#2586] [功能建议 / Ideas]
![微信图片_20211205222244](https://user-images.githubusercontent.com/61007431/144750526-8d884473-639f-493a-b987-f109ecb50351.png)

**Comment by the1812** [comment]
这不就是 #2585?

**Reply by LOVE-END** [reply]
> 这不就是#2585？

噢噢,对不起对不起,家里网上GitHub太卡了,放错图片了
应该是这个
![微信图片_20211206002331](https://user-images.githubusercontent.com/61007431/144755170-dd1e5035-1d47-49dc-b077-a422d443573b.png)
